proto/frequenz/api/dispatch/v1/dispatch.proto

```diff
@@ -118,13 +115,28 @@ message Dispatch {
   // actually affecting any component states.
   // Notably, a dispatch can be both "dry run" and "active," allowing for
   // the system to generate logs and observe behavior without making actual changes.
-  bool is_dry_run = 10;
+  bool is_dry_run = 6;
 
   // The dispatch payload
-  google.protobuf.Struct payload = 11;
+  google.protobuf.Struct payload = 7;
 
   // The recurrence rule
-  RecurrenceRule recurrence = 12;
+  RecurrenceRule recurrence = 8;
+}
+
+// Represents a dispatch, including its metadata
+message DispatchDetail {
```

Comment:
Bike shed: what about having a `DispatchData`, `DispatchMetadata`, and then `Dispatch` is a message containing both? Not convinced but this details object sounds confusing to me. Or maybe just rename this one `DispatchWithMetadata`?

Reply:
That was my first draft as well, but apparently there was a cross-api-project decision and all should follow that pattern, e.g. https://github.com/frequenz-floss/frequenz-api-electricity-trading/blob/v0.x.x/proto/frequenz/api/electricity_trading/v1/electricity_trading.proto

I personally prefer that kind of separation as well..